.codespellrc

```diff
@@ -0,0 +1,10 @@
+# https://github.com/codespell-project/codespell
+[codespell]
+builtin = clear,rare,informal
+check-filenames =
+check-hidden =
+ignore-words = .codespellignore
+interactive = 1
+skip = .git,go.mod,go.sum,CHANGELOG.md,LICENSES,venv
```

Comment:
Can we add `internal/include/libbpf` here? It was mentioned in the last SIG meeting that that directory is just copied from upstream. Fixing upstream spelling errors here will just result in thrash.

cc @grcevski

Reply:
Related: https://github.com/libbpf/libbpf/pull/819

Reply:
Fixed